addons/mail/static/src/core/common/composer.js

```diff
@@ -113,6 +113,7 @@ export class Composer extends Component {
             active: true,
             isFullComposerOpen: false,
         });
+        this.root = useRef("root");
```

Comment:
I don't see where this is used in the diff?

Reply:
It's very odd, but the overlay service assumes there is a root property on the component passed in the context. So it's related to the new context passed to `dialog.add`